apps/agent-portal/backend/README.md

```
@@ -0,0 +1,82 @@
+| Variable | Description |
+|---|---|
+| `ENTITY_BASE_URL` | Base URL of the entity service |
+| `ENTITY_TOKEN_URL` | OAuth2 token endpoint |
+| `ENTITY_CLIENT_ID` | OAuth2 client ID |
+| `ENTITY_CLIENT_SECRET` | OAuth2 client secret |
+| `ENTITY_SCOPES` | Comma-separated OAuth2 scopes (optional) |
+| `PORT` | Server listen address (default `:8080`) |
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)